src/qt/createwalletdialog.cpp

```diff
@@ -34,6 +34,11 @@ CreateWalletDialog::CreateWalletDialog(QWidget* parent) :
             ui->disable_privkeys_checkbox->setChecked(false);
         }
     });
+
+#ifndef USE_SQLITE
+    ui->descriptor_checkbox->setToolTip(tr("Compiled without sqlite support (required for descriptor wallets)"));
+    ui->descriptor_checkbox->setEnabled(false);
```

Comment:
I think this should also do `setChecked(false)`. When descriptor wallets become the default and this is default checked, that will be necessary.